src/Storage/Service/AbstractUrlStorage.php

```diff
@@ -185,8 +185,8 @@ public function isReadOnly(): bool
         return $this->readOnly;
     }
 
-    public function isToSkip(): bool
+    public function isSkip(): bool
     {
-        return $this->toSkip;
+        return $this->skip;
```

Comment:
```suggestion
        return $this->shouldSkip;
```

I wonder if its not better to also name the variable with "is" or "should" prefix.

private $readOnly; indicates less then private $isReadOnly; that we are handling a bool.
Idem for private $skip vs private $shouldSkip;